config/envConfig.ts

```diff
@@ -0,0 +1,24 @@
+import dotenv from 'dotenv';
+dotenv.config();
+import { development } from './development.js';
```

Comment:
If you chnage the env to the way i said you should do it without prefixing the env, it would fix this, you wont have to import this one after the other

Reply:
Oh. I did not really get how you asked me to do it. You mentioned you will discuss it later. So I just did this my way hoping it fixes the issue.